docs/deploy-configure.md

```diff
@@ -24,7 +25,52 @@ To deploy your first replica set:
    ```
    mongo "mongodb://<service-object-name>.<namespace>.svc.cluster.local:27017/?replicaSet=<replica-set-name>"
    ```
-<em>NOTE: You can access the mongodb instance only from within a pod running in the cluster.</em>
+<em>NOTE: You can access the mongodb instance only from within a pod
```

Comment:
Any reason to use "mongodb" instead of "MongoDB" here? Are we talking about a MongoDB instance or a k8s instance named "mongodb"?

Reply:
updated. @chatton should this be `mongod` instead?

Reply:
yes @jwilliams-mongo! `mongod`  is more correct here.

Reply:
@chatton updated